Gems/Multiplayer/Code/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ ly_add_target(
             AZ::AzCore
             AZ::AzFramework
             AZ::AzNetworking
+            Legacy::CryCommon
```

Comment:
We shouldn't be adding a dependency on Legacy library such as CryCommon.

Reply:
Updated! Added a bus to AzFramework.